src/daemon/org.projectatomic.rpmostree1.xml

```diff
@@ -393,6 +393,12 @@
       <arg type="s" name="transaction_address" direction="out"/>
     </method>
 
+    <!-- Experimental generic Search -->
+    <method name="Search">
+      <arg type="as" name="query" direction="in"/>
+      <arg type="a{sv}" name="options" direction="in"/>
```

Comment:
Fine as is, but since this is experimental we can probably also not have an `options` field, i.e. there's no concern with changing the API yet.